futures-await-async-macro/src/lib.rs

```diff
@@ -293,14 +293,14 @@ pub fn async_move(attribute: TokenStream, function: TokenStream) -> TokenStream
                 ::futures::__rt::std::boxed::Box<::futures::__rt::Future<
                     Item = <! as ::futures::__rt::IsResult>::Ok,
                     Error = <! as ::futures::__rt::IsResult>::Err,
-                > + ::futures::__rt::pin_api::Unpin + #(#lifetimes +)*>
+                > + #(#lifetimes +)*>
             }
         } else if boxed && send {
             quote_cs! {
                 ::futures::__rt::std::boxed::Box<::futures::__rt::Future<
                     Item = <! as ::futures::__rt::IsResult>::Ok,
                     Error = <! as ::futures::__rt::IsResult>::Err,
-                > + ::futures::__rt::pin_api::Unpin + Send + #(#lifetimes +)*>
```

Comment:
Why was the `Unpin` bound removed?

Reply:
I assumed that this was needed for the [old `impl<T: Future + Unpin> StableFuture for T`](https://github.com/withoutboats/futures-stable/blob/2819fb44131e42e16e0e923a9bf681b197ae01ed/src/lib.rs#L50), so now that there is `impl<T: Future> StableFuture for T` it is not required anymore. I guess it's still actually correct to specify it even if it's not necessary though.

Reply:
I'd like to specify it so that downstream users can rely on it. Its why they tagged them `async_move` after all.